src/config.js

```diff
@@ -32,7 +32,7 @@ const sortSearchByDistance = false;
 //
 // In a way, 'processAlias' defines which transaction process (or processes)
 // this particular web application is able to handle.
-const bookingProcessAlias = 'preauth-nightly-booking/release-1';
+const bookingProcessAlias = 'privileged-custom-pricing/release-1';
```

Comment:
Is this the new process name we've chosen @sktoiva? It puts a lot of emphasis on the privileged + custom pricing parts. Those are the new thing so it makes sense in this moment of time but if we'd come out the gate with this process then would we call it that or instead just something like flex-default-process/release-1? There's no more one defining characteristic for this process like before when it was the pricing. Now it's just a default starting point that supports many kind of pricing structures but also encodes things like double blind reviews and some choices about possibility to cancel and notification emails. We won't get too many chances to rename the process so worth thinking this through. I'd like to hear your thoughts before we commit @sktoiva @kpuputti @lyyder.

Reply:
Oh, I should've added a comment here. This is just the name I put to the new process in our staging Saunatime marketplace. We will need to decide the new process name here or later, but This PR will most likely undo this change and release a version of FTW that uses the new line items endpoint for the estimation, but otherwise keep the current default process with its non-privileged transitions.

Then we can easily enable privileged transitions by marking some of the transitions in `util/transactions.js` as privileged, and at some point make that the default.

Does this make sense?

Reply:
Summary: the new process will be `flex-default-process`, but that will be taken into use in another PR. Core is already using that for recent marketplaces that have the process with privileged transitions added.